packages/dds/tree/src/test/utils.ts

```diff
@@ -113,9 +113,9 @@ export class TestTreeProvider {
      * @param index - the container that will spawn the summarizer client
      * @returns a function which will cause a summary to happen when awaited. May be called multiple times.
      */
-    public async enableManualSummarization(index = 0): Promise<() => Promise<void>> {
-        assert(index < this.trees.length, "Index out of bounds: not enough trees");
-        const summarizer = await createSummarizer(this.provider, this.containers[index]);
+    public async enableManualSummarization(): Promise<() => Promise<void>> {
+        assert(this.trees.length === 1, "Manual summarization cannot be enabled for multiple trees.")
```

Comment:
It would be good to document what the issue preventing this is (ex: a todo comment like `// TODO: support manual summarization with multiple trees once bug #???? is fixed.`.)

Reply:
`enableManualSummarization()` method removed